src/pages/migrating/guide/develop.mdx

```diff
@@ -17,24 +17,23 @@ Step-by-step guide to updating your code from Carbon v10 to v11.
 <AnchorLinks>
 
 <AnchorLink>Overview</AnchorLink>
-<AnchorLink>carbon-components-react</AnchorLink>
-<AnchorLink>carbon-components</AnchorLink>
-<AnchorLink>@carbon/icons-react</AnchorLink>
-<AnchorLink>carbon-icons</AnchorLink>
-<AnchorLink>Elements</AnchorLink>
+<AnchorLink>Migrating a React app</AnchorLink>
```

Comment:
the link in this notification needs to be updated 
<img width="777" alt="image" src="https://github.com/carbon-design-system/carbon-website/assets/40550942/ce4cf816-b23a-45b9-80ff-a07c74ec30c5">

Reply:
on it!